build/generate-docs.ts

```diff
@@ -517,6 +518,11 @@ function createExpressionsContent() {
     fs.writeFileSync(`${BASE_PATH}/expressions.md`, content);
 }
 
+function createSchemaContent() {
+    const content = fs.readFileSync('build/schema.md', 'utf-8');
```

Comment:
Is this really needed? We have other .md files in the docs folder...

Reply:
I've updated this a little - now v8 description is concatenated with a partial describing the state structure.